README.md

```diff
@@ -2,8 +2,8 @@
 
 # Essential Next.js Build Plugin (beta)
 
-
-:warning: This is the beta version of the Essential Next.js plugin. For the stable version, see [Essential Next.js plugin v3](https://github.com/netlify/netlify-plugin-nextjs/tree/v3#readme) :warning:
+:warning: This is the beta version of the Essential Next.js plugin. For the stable version, see
```

Comment:
Minor cleanup to avoid a vision-based term:

```suggestion
:warning: This is the beta version of the Essential Next.js plugin. For the stable version, refer to
```